lib/oci_utils/impl/migrate/oci-image-migrate-import-main.py

```diff
@@ -25,10 +25,7 @@
 from oci_utils.migrate import system_tools
 from oci_utils.migrate import terminal_dimension
 from oci_utils.migrate.exception import OciMigrateException
-from oci_utils.migrate.migrate_tools import get_config_data as get_config_data
-
-if sys.version_info.major < 3:
-    exit_with_msg('Python version 3 is a requirement to run this utility.')
+from oci_utils.migrate.migrate_tools import get_config_data
 
```

Comment:
if 'ci' is kind of sub-category of "importation". better to use oci-utils.import.ci as logger name

Reply:
right, this is a mess
changed